WayMovies/View/FavoritesCollection.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  FavoritesCollection.swift
+//  WayMovies
+//
+//  Created by Hersh Nagpal on 1/29/20.
+//  Copyright © 2020 Hersh Nagpal. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+var favoritesMap = [Int:TVItem]()
```

Comment:
? - it seems odd to have a global variable that sort of hangs out in a limbo. You may run into race condition and/or threading issues if this is accessed a lot